jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -291,6 +318,9 @@ async def process(app: Flask, qsm: QueueService):  # pylint: disable=redefined-o
 
         for batch_processing in batch_processings:
             await processor.process(batch_processing)
+        processor.generate_paper_letters()
+        # TODO: send AR and transition pdf to BCMail+
+        # TODO: mark MAIL entries as PROCESSED/FAILED
```

Comment:
Before this PR, mail furnishing entries are marked as `PROCESSED`. Since SFTP to BCMail+ will do some basic error handling, I assume that there will be some tweaks and leave the entries in `QUEUED` status for now.